run-17495340446-artifacts/scripts/style_normalize.py

```
@@ -0,0 +1,230 @@
+
+def replace_banned_phrases(s: str) -> str:
+    """Replace banned phrases with preferred alternatives."""
+    result = s
+    for banned, replacement in BANNED_PHRASES.items():
+        # Case-insensitive replacement preserving original case pattern
+        pattern = re.compile(re.escape(banned), re.IGNORECASE)
+        
+        def replace_match(match):
+            original = match.group(0)
+            if original.isupper():
+                return replacement.upper()
```

Ruff (0.12.2)

Function definition does not bind loop variable `replacement`

(B023)